### PR TITLE
Update Modbus registers for S1155 based on firmware 2.26.3.

### PR DESCRIPTION
**Pull request by HusoLodge**
I followed the steps explained:
- Changed language to English on heat pump
- Exported all registers
- Merged the changes to  s1155_s1255.csv
- Convert source CSV files to JSON using the script provided

I have also attached the original export file:
[modbus_addresses_all_20240327-06544322332001-english.csv](https://github.com/yozik04/nibe/files/14786311/modbus_addresses_all_20240327-06544322332001-english.csv)

### CODE REVIEW

File: nibe/data/s1155_s1255.csv [hunk 1]
```diff
@@ -1131,13 +1144,13 @@ Max difference, SES lower priority energy source	MODBUS_HOLDING_REGISTER	1327	10
...
 EME20	MODBUS_HOLDING_REGISTER	4000	1		4	0	1	0
-Current power (EME 20)	MODBUS_INPUT_REGISTER	2176	1	W	6	0	0	0
+Current power	MODBUS_INPUT_REGISTER	2176	1	W	6	0	0	0
 Speed of circulation pump passive cooling (EP15)	MODBUS_HOLDING_REGISTER	4006	1	%	4	1	100	70
 Speed of circulation pump active cooling (EP15)	MODBUS_HOLDING_REGISTER	4007	1	%	4	1	100	70
 Ground water pump's control signal	MODBUS_HOLDING_REGISTER	4012	1		1	0	0	0
 BT12 offset	MODBUS_HOLDING_REGISTER	4014	10	°C	1	-50	50	0
-Total average power (EME 20)	MODBUS_INPUT_REGISTER	2178	1	W	6	0	0	0
-Total energy (EME 20)	MODBUS_INPUT_REGISTER	2180	10	kWh	3	0	0	0
+Total average power (EME 20)	MODBUS_INPUT_REGISTER	2178	100	kW	6	0	0	0
```
Comment (yozik04) [inline nibe/data/s1155_s1255.csv:1152]: Factor 100 seems to be too high. Interesting if it is correct.

File: nibe/data/s1155_s1255.csv [hunk 2]
```diff
@@ -1171,45 +1184,56 @@ id:6920	MODBUS_HOLDING_REGISTER	4074	1		4	0	20	0
...
-Minimum permitted speed (EB100 GP1)	MODBUS_HOLDING_REGISTER	3242	1	%	4	1	50	1
-id:8060	MODBUS_HOLDING_REGISTER	3259	1		1	0	1	0
-id:8061	MODBUS_HOLDING_REGISTER	3260	1		1	0	1	0
-id:8062	MODBUS_HOLDING_REGISTER	3261	1		1	0	1	0
-id:8063	MODBUS_HOLDING_REGISTER	3262	1		1	0	1	0
-id:8064	MODBUS_HOLDING_REGISTER	3263	1		1	0	1	0
-id:8065	MODBUS_HOLDING_REGISTER	3264	1		1	0	1	0
-id:8066	MODBUS_HOLDING_REGISTER	3265	1		1	0	1	0
-id:8067	MODBUS_HOLDING_REGISTER	3266	1		1	0	1	0
+Minimum permitted speed (EB100 GP1)	MODBUS_HOLDING_REGISTER	5351	1	%	4	1	50	1
```
Comment (yozik04) [inline nibe/data/s1155_s1255.csv:1228]: @elupus I do not understand why these register addresses differ so much. Maybe we need to split some models?

File: nibe/data/s1155_s1255.csv (hunk 1 above)
Comment (elupus) [reply nibe/data/s1155_s1255.csv:1152]: Seen same for F series. So i think its correct.

File: nibe/data/s1155_s1255.csv (hunk 2 above)
Comment (elupus) [reply nibe/data/s1155_s1255.csv:1228]: No idea. But they are unnamed registers soni think we can ignore them for the most part. Dont think we need to split.

**Comment by yozik04** [comment]
Thank you @HusoLodge!
I do not plan to create a release yet. We will bundle it with other changes when they come.

**Pull request merged**